### DISCUSSION TITLE
Duvida Sobre aula: Calculator

### DISCUSSION THREAD

**Discussion by Alexmau10** [balta-io/2801#78] [Perguntas e Respostas] [answered]
Prezados, 
Por favor podem me dizer por que estou com erro no "void" ? E na verdade o erro já acontece desde o using...

![image](https://user-images.githubusercontent.com/89946503/168487616-50c2c16a-c5ad-4a8b-89cd-54a561613c57.png)

**Comment by andrebaltieri** [comment, marked as answer]
@Alexmau10 você escreveu `Static` ao invés de `static`. Conforme mostrado nas aulas, o C# é case sensitive, o que significa que ele diferencia minúsculas de maiúsculas.

**Reply by Alexmau10** [reply]
Perfeito! @andrebaltieri - case sensitive.
Grato!

**Reply by andrebaltieri** [reply]
💜
